test/unit/CMakeLists.txt

```diff
@@ -6,8 +6,8 @@ set(source
    )
 
 list(APPEND source ${cse_common_source})
-
 add_executable(cse_tests ${source})
+target_compile_definitions(cse_tests PRIVATE XIOPAK_UNIT_PATH="${CMAKE_SOURCE_DIR}/test/unit/xiopak.unit.cpp")
```

Comment:
See my response to the other thread. Let me know if you want to talk through it.